torchtext/experimental/datasets/__init__.py

```diff
@@ -6,23 +6,25 @@
 from .translation import Multi30k, IWSLT, WMT14
 from .question_answer import SQuAD1, SQuAD2
 
-__all__ = ['LanguageModelingDataset',
-           'WikiText2',
-           'WikiText103',
-           'PennTreebank',
-           'WMTNewsCrawl',
-           'IMDB',
-           'AG_NEWS',
-           'SogouNews',
-           'DBpedia',
-           'YelpReviewPolarity',
-           'YelpReviewFull',
-           'YahooAnswers',
-           'AmazonReviewPolarity',
-           'AmazonReviewFull',
-           'UDPOS',
-           'CoNLL2000Chunking',
-           'Multi30k',
-           'IWSLT',
-           'WMT14',
-           'SQuAD1', 'SQuAD2']
+DATASETS = {'WikiText2': WikiText2,
+            'WikiText103': WikiText103,
+            'PennTreebank': PennTreebank,
+            'WMTNewsCrawl': WMTNewsCrawl,
+            'IMDB': IMDB,
+            'AG_NEWS': AG_NEWS,
+            'SogouNews': SogouNews,
+            'DBpedia': DBpedia,
+            'YelpReviewPolarity': YelpReviewPolarity,
+            'YelpReviewFull': YelpReviewFull,
+            'YahooAnswers': YahooAnswers,
+            'AmazonReviewPolarity': AmazonReviewPolarity,
+            'AmazonReviewFull': AmazonReviewFull,
+            'UDPOS': UDPOS,
+            'CoNLL2000Chunking': CoNLL2000Chunking,
+            'Multi30k': Multi30k,
+            'IWSLT': IWSLT,
+            'WMT14': WMT14,
+            'SQuAD1': SQuAD1,
+            'SQuAD2': SQuAD2}
+
+__all__ = ['LanguageModelingDataset'] + list(map(str, DATASETS.keys()))
```

Comment:
Not sure if we still want to expose `LanguageModelingDataset` to users.

Reply:
I'll make that a separate PR.